fix/fixtures/pass/0001_with_raw_backticks.anko.fizz

```diff
@@ -8,9 +8,18 @@ create_table("users", func(t) {
   t.Column("joined_at", "timestamp", {})
 })
 
-create_table("domains") {
+create_table("domains", func(t) {
+  t.Column("id","uuid",{"primary":true})
+})
+
+create_table("sodas", func(t) {
   t.Column("id","uuid",{"primary":true})
 }
+)
```

Comment:
I'm clearly just missing it since the tests are passing, but at some point you are  going to have to explain to me how it's catching this straggler seeing as it looks to me as though it would have moved out of the create_table logic the line before.